sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1994,8 +1994,7 @@ def hpat_pandas_series_sum(
     .. only:: developer
 
         Tests:
-            python -m sdc.runtests sdc.tests.test_series.TestSeries.test_series_sum1
-            # python -m sdc.runtests sdc.tests.test_series.TestSeries.test_series_sum2
+            python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_sum
```

Comment:
Need to refer to all tests for sum method:
python -m sdc.runtests -k sdc.tests.test_series.TestSeries.test_series_sum*